### PR TITLE
es 5 upgrade india

### PR DESCRIPTION
**Pull request by shyamkumarlchauhan**
https://dimagi-dev.atlassian.net/browse/SAAS-14657

##### Environments Affected
india

### CODE REVIEW

File: environments/india/public.yml [hunk 1]
```diff
@@ -7,8 +7,8 @@ DATADOG_ENABLED: True
 elasticsearch_endpoint: '{{ groups.elasticsearch.0 }}'
 #elasticsearch_memory: '4096m'
 elasticsearch_cluster_name: 'indiahqes-1.x'
```
Comment (AmitPhulera) [inline environments/india/public.yml:9]: Should we change the `elasticsearch_cluster_name` too?

Comment (AmitPhulera) [reply environments/india/public.yml:9]: Not really sure, what all will it impact.

Comment (shyamkumarlchauhan) [reply environments/india/public.yml:9]: Not sure it is a good idea at this point of time.

**Comment by shyamkumarlchauhan** [comment]
Changes already applied in another PR.

**Pull request closed without merging**